locale/cs_CZ/foreman_azure_rm.po

```diff
@@ -0,0 +1,183 @@
+# Czech translations for foreman_azure_rm package.
+# Copyright (C) 2020 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the foreman_azure_rm package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: foreman_azure_rm 1.0.0\n"
```

Comment:
Why does this have version as `1.0.0` at many places? At few places there is correct version `2.0.8`.

Reply:
I corrected the version details, i think it happened because we were not having any translations pulled for this project and first time when it pull the po files were having old version. About the new version of azure, when you will bump the version of the project, I mean changing it in version file, I will again update the pr, and then we merge. So maybe I will just squash the commits at the end.